sdk/core/Azure.Core/src/Operation.cs

```diff
@@ -64,7 +65,8 @@ public abstract class Operation
         /// <remarks>
         /// This method will periodically call UpdateStatusAsync till HasCompleted is true, then return the final response of the operation.
         /// </remarks>
-        public abstract ValueTask<Response> WaitForCompletionResponseAsync(CancellationToken cancellationToken = default);
+        public virtual ValueTask<Response> WaitForCompletionResponseAsync(CancellationToken cancellationToken = default)
```

Comment:
Any reason not to inline the DefaultWaitForCompletionResponseAsync implementation?